### ISSUE TITLE
Can this handle "restricted" accounts?

### ISSUE THREAD

**Issue by DonaldTsang** [PJHRobles/Twitter-Get-Old-Tweets-Scraper#1]
https://twitter.com/Susan_Texan or https://twitter.com/DoubleTMisterB are both inaccessible to public searches unless a user logs in. Is it possible to create a workaround?

**Comment by PJHRobles** [comment]
No, this cannot handle restricted accounts. I don't think any restricted accounts are accessible to scrapers unless you login with a user that follows that account.

**Comment by DonaldTsang** [comment]
@PJHRobles It is not required to follow the account, but it is required to turn of the "filter" so that the logged in browser can access "restricted" accounts
